war/src/main/js/pluginSetupWizardGui.js

```diff
@@ -913,7 +913,7 @@ var createPluginSetupWizard = function(appendTarget) {
 								if (j.correlationId) {
 									selectedPluginNames.push(j.name);
 								}
-								setFailureStatus(j)
+								setFailureStatus(j);
```

Comment:
@oleg-nenashev This PR has the goal to detect this kind of bugs. This bug was introduced recently (after my PR), we need to solve it but the build will fail.